tools/DevDiagnostics/DevHome.DevDiagnostics/NativeMethods.txt

```diff
@@ -1,119 +1,120 @@
-IsWindow
-IsWindowArranged
-IsWindowVisible
-IsWow64Process2
-IsZoomed
-LoadIcon
-LoadImage
-MessageBox
-MonitorFromWindow
-MoveWindow
-NtQueryInformationProcess
-OpenClipboard
-OpenProcessToken
-RegisterHotKey
-RemoveClipboardFormatListener
-RmRegisterResources
-RmStartSession
-RmGetList
-RmEndSession
-SendMessage
-SetFocus
-SetForegroundWindow
-SetParent
-SetWindowLong
-SetWindowLong
-SetWindowPos
-SetWindowRgn
-SetWindowSubclass
-SetWinEventHook
-ShowWindow
-SystemParametersInfo
-UnhookWinEvent
-UnregisterHotKey
-WaitForSingleObject
-WindowFromPoint
-CF_*
-DEVICE_SCALE_FACTOR
-DWMWA_*
-DWMWINDOWATTRIBUTE
-EVENT_*
-EVENT_OBJECT_DESTROY
-EVENT_OBJECT_LOCATIONCHANGE
-EVENT_SYSTEM_FOREGROUND
-GET_ANCESTOR_FLAGS
-GET_CLASS_LONG_INDEX
-HMONITOR
-HWND_*
-ICON_*
-ICON_*
-ICON_*
-IDI_*
-IMAGE_*
-INVALID_HANDLE_VALUE
-LR_*
-MOD_*
-// MONITOR_DEFAULTTONEAREST
-PROCESS_BASIC_INFORMATION
-PROCESS_INFORMATION_CLASS
-PROCESS_MACHINE_INFORMATION
-RM_APP_TYPE
-RM_UNIQUE_PROCESS
-RM_PROCESS_INFO
-RM_REBOOT_REASON
-SECTION_FLAGS
-SUBCLASSPROC
-SW_*
-VIRTUAL_KEY
-WIN32_ERROR
-WINEVENT_*
-WINEVENT_OUTOFCONTEXT
-WINEVENT_SKIPOWNPROCESS
-WINDOWPOS
-WM_*
-WS_*
+AddClipboardFormatListener
+CallWindowProc
+CloseClipboard
+CoCreateInstance
+CombineRgn
+CreateEllipticRgn
+CreateRectRgn
+CreateRoundRectRgn
+DefSubclassProc
+DwmGetWindowAttribute
+DwmSetWindowAttribute
+DwmSetWindowAttribute
+EnumChildWindows
+EnumThreadWindows
+EnumWindows
+GetAncestor
+GetBinaryType
+GetClassLong
+GetClassName
+GetClipboardData
+GetCurrentPackageFullName
+GetCursorPos
+GetDesktopWindow
+GetForegroundWindow
+GetMonitorInfo
+GetMonitorInfo
+GetOpenFileName
+GetParent
+GetProcessInformation
+GetScaleFactorForMonitor
+GetTopWindow
+GetWindowInfo
+GetWindowLong
+GetWindowRect
+GetWindowText
+GetWindowText
+GetWindowTextLength
+GetWindowTextLength
+GetWindowThreadProcessId
+GlobalAddAtom
+GlobalDeleteAtom
+GlobalLock
+GlobalUnlock
+IsIconic
+IsImmersiveProcess
+IsWindow
+IsWindowArranged
+IsWindowVisible
+IsWow64Process2
+IsZoomed
+LoadIcon
+LoadImage
+MessageBox
+MonitorFromWindow
+MoveWindow
+NtQueryInformationProcess
+OpenClipboard
+OpenProcessToken
+RegisterHotKey
+RemoveClipboardFormatListener
+RmRegisterResources
+RmStartSession
+RmGetList
+RmEndSession
+SendMessage
+SetFocus
+SetForegroundWindow
+SetParent
+SetWindowLong
+SetWindowLong
+SetWindowPos
+SetWindowRgn
+SetWindowSubclass
+SetWinEventHook
+ShowWindow
+SystemParametersInfo
+UnhookWinEvent
+UnregisterHotKey
+WaitForSingleObject
+WindowFromPoint
+CF_*
+DEVICE_SCALE_FACTOR
+DWMWA_*
+DWMWINDOWATTRIBUTE
+EVENT_*
+EVENT_OBJECT_DESTROY
+EVENT_OBJECT_LOCATIONCHANGE
+EVENT_SYSTEM_FOREGROUND
+GET_ANCESTOR_FLAGS
+GET_CLASS_LONG_INDEX
+HMONITOR
+HWND_*
+ICON_*
+ICON_*
+ICON_*
+IDI_*
+IMAGE_*
+INVALID_HANDLE_VALUE
+LR_*
+MOD_*
+// MONITOR_DEFAULTTONEAREST
+PROCESS_BASIC_INFORMATION
+PROCESS_INFORMATION_CLASS
+PROCESS_MACHINE_INFORMATION
+RM_APP_TYPE
+RM_UNIQUE_PROCESS
+RM_PROCESS_INFO
+RM_REBOOT_REASON
+SECTION_FLAGS
+STATUS_*

```

Comment:
Why all of them?

Reply:
Turns out I didn't need this. Must have been from some earlier experimentation.